lib/Models/TableDataSource.js

```diff
@@ -326,7 +326,6 @@ function setOneFeaturePerRow(dataSource, tableStructure, specialColumns) {
     // If there more entities already exist than there are in the table,
     // remove the extras.
     var entities = dataSource._entityCollection;
-    entities.suspendEvents();
```

Comment:
Why was it necessary to remove these lines?  I have to imagine it makes loading a CSV much slower i some scenarios.